packages/design-system/src/components/FilterChip/FilterChip.tsx

```diff
@@ -81,6 +78,9 @@ export class FilterChip extends React.Component<FilterChipProps> {
         className={buttonClassNames}
         onClick={this.handleClick}
         onKeyDown={this.handleKeyDown}
+        aria-label={`${label} . ${
+          ariaClearLabel ?? t('filterChip.ariaClearLabel')
+        } ${label} filter`}
```

Comment:
does the word 'filter' need to be localized too?

Reply:
Ah, yep, good catch

Reply:
The word order might even be different in Spanish for the phrase "_something_ filter"

Reply:
Got translations for this and updated the code